entities/bombStatus.entity.ts

```diff
@@ -0,0 +1,36 @@
+import { BaseEntity, Column, CreateDateColumn, Entity, ManyToOne, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { BombStatusChange } from 'types/BombStatusChange.enum';
+
+import { PlayerInfo } from './playerInfo.entity';
+import { Position } from './position.entity';
+import { Round } from './round.entity';
+
+/**
+ * Database record
+ */
+@Entity()
+export class BombStatus extends BaseEntity {
+  /**
+   * Primary key
+   */
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @CreateDateColumn()
+  createdAt: Date;
```

Comment:
What is this field?

Reply:
Datestamp when the record is created. Mostly useful for backend stuff, I don't know if you will benefit from this in the frontend